cmd/bblfsh/client.go

```diff
@@ -39,10 +40,19 @@ func (c *clientCmd) Execute(args []string) error {
 		run = c.runStandalone
 	}
 
+	var encoding protocol.Encoding
+	switch(c.Encoding) {
+	case "Base64":
+		encoding = protocol.Base64
+	default:
+		encoding = protocol.UTF8
```

Comment:
Well this cover the switch case for "UTF8" and the "no encoding specified" and since the default is UTF8 anyway it's the same result with an ugly empty or missing `default` less.